index.html

```diff
@@ -12,18 +12,24 @@
 </head>
 
 <body>
-    <header>
-        <div class="top-page">
-            <div class="ornament-top"><img src="/pict-mobile/image_geometry_1.svg" alt=""></div>
-            <nav class="menu1"><img src="/pict-mobile/ic_menu.svg" alt="Menu icon"></nav>
-            <div class="header-desktop">
-                <div>Hello</div>
-                <div>Portfolio</div>
-                <div>About</div>
-                <div>Contact</div>
-            </div>
+      <!-- code by abubakar -->
+    <header class="header active">
+        <img src="logo" alt="logo" class="logo">
+        <nav class="navbar">
+            <ul class="navbar-list">
+                <li><a class="navbar-link" href="#">Hello</a></li>
+                <li><a class="navbar-link" href="#">Portfolio</a></li>
+                <li><a class="navbar-link" href="#">About</a></li>
+                <li><a class="navbar-link" href="#">Contact</a></li>
```

Comment:
- [ ] Hi you did great but as part of the requirements, each nav link is supposed to direct to a specific section on the page and in the mobile menu, when clicking on a link its supposed to close the mobile menu and redirect you to a specific section of the page

i.e to add the link that redirects to the contact form, you have to specify the path `<li><a class="navbar-link" href="#contact">Contact</a></li>`, and add the id name to that specific section `<section class="foot-section" id="contact">`